wordpress-importer.php

```diff
@@ -787,6 +787,7 @@ function process_posts() {
 						if ( isset( $inserted_comments[$comment['comment_parent']] ) )
 							$comment['comment_parent'] = $inserted_comments[$comment['comment_parent']];
 						$comment = wp_filter_comment( $comment );
+						$comment = wp_slash( $comment );
```

Comment:
@ocean90 thanks for reviewing. I think I've fixed that, moving the wp_slash up one line.